<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1060</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>1341</width>
      <height>951</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="XyPlot" name="xyPlot_Ch1" native="true">
       <property name="styleSheet">
        <string notr="true">color: rgb(56, 162, 255);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XyPlot" name="xyPlot_Ch2" native="true">
       <property name="styleSheet">
        <string notr="true">color: rgb(56, 162, 255);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XyPlot" name="xyPlot_Ch3" native="true">
       <property name="styleSheet">
        <string notr="true">color: rgb(56, 162, 255);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XyPlot" name="xyPlot_Ch4" native="true">
       <property name="styleSheet">
        <string notr="true">color: rgb(56, 162, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="buttonMainClose">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Main Breaker Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSsrClose">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SS Resistor Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttoMainOpen">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Main Breaker Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSsrOpen">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SS Resistor Open</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="pictures.qrc">
     <normaloff>:/Icons/Play.png</normaloff>:/Icons/Play.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to serial port</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="pictures.qrc">
     <normaloff>:/Icons/Stop.png</normaloff>:/Icons/Stop.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from serial port</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="pictures.qrc">
     <normaloff>:/Icons/Settings.png</normaloff>:/Icons/Settings.png</iconset>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
   <property name="toolTip">
    <string>Serial port settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>XyPlot</class>
   <extends>QWidget</extends>
   <header location="global">xyplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="pictures.qrc"/>
 </resources>
 <connections/>
</ui>
